services/api/src/routes/game.ts

```diff
@@ -3,16 +3,19 @@ import Joi from 'joi';
 
 const router = express.Router();
 
+const validationOfPath = () => {
+  const uuidPath = '671e2367-86c3-453a-9df8-3c0048145b64';
```

Comment:
issikeliam sita uuid i .env faila. Cia turbut bus `ROOM_UUID`?

Reply:
yes, bet kaip bus del pleciamumo, nes cia tik initial sprendimas tureti viena room, o paskui daugiau?

Reply:
Sutinku, kad turbūt šitam dalykui env failas nėra būtinas, bet spėju, kad be .env failo vis tiek neapsisuksim, tai sakyčiau, kad nepatrukdys jį introducinti anksčiau. Taip pat, kol dirbam su vienu roomu, turbūt dažnai teks referint į šitą uuid, tai tuo pačiu turėsim vietą, kur galėsim greit atrast jį.

Reply:
done